projects/ontimize-web-ngx/src/lib/components/list/o-list.component.ts

```diff
@@ -428,4 +428,11 @@ export class OListComponent extends AbstractOServiceComponent<OListComponentStat
     ObservableWrapper.callEmit(this.onDataLoaded, this.dataResponseArray);
   }
 
+  public registerQuickFilter(arg: any): void {
+    super.registerQuickFilter(arg);
+    if (Util.isDefined(this.quickFilterComponent) && Util.isDefined(this.state.quickFilterActiveColumns)) {
```

Comment:
Can we ensure that _this.state_ at this point is initialized? Maybe it should be better to add a checking:
```
... && Util.isDefined(this.state) &&
```